src/iosched/unified.c

```diff
@@ -1117,7 +1193,8 @@ ltfs_thread_return _unified_writer_thread(void *iosched_handle)
 		ltfs_profiler_add_entry(priv->profiler, &priv->proflock, IOSCHED_REQ_ENTER(REQ_IOS_IOSCHED));
 		if (! priv->writer_keepalive) {
 			ltfs_thread_mutex_unlock(&priv->queue_lock);
-			_unified_flush_all(priv);
+			ltfs_thread_cond_broadcast(&priv->writer_cond);
+			_unified_exit_all(priv);
```

Comment:
You have other changes where instead of removing an existing call to `_unified_flush_all` you keep it around, but conditionally building that part of the code depending on `USE_DIRECT_FLUSH` being set or not. Perhaps you'd like to do the same here?

Reply:
Previous `_unified_flush_all` is current `_unified_exit_all`.

In this point I want to call previous `_unified_flush_all` in all cases. So I believe conditional compile is not needed.